blackbox/docs/best_practice/full_restart_upgrade.txt

```diff
@@ -0,0 +1,175 @@
+.. highlight:: sh
+
+.. _full_restart_upgrade:
+
+====================
+Full Restart Upgrade
+====================
+
+The preferred way to upgrade a running CrateDB cluster is via the
+:ref:`rolling_upgrade` method. But this is not always an option (for example,
+when you are upgrading from one minor version to the next).
```

Comment:
would write this differently. so that only patch versions can be rollings. "minor" sounds small and therefore confusing for people not aware that this is actually the feature version. (or call it feature version)